[MeL] Changes in MeshLib used for GeoMapper refactorization.

TomFischer opened the pull request:
See commit messages for details.

norihiro-w commented on MeshLib/MeshSearch/MeshElementGrid.h:
```diff
@@ -72,6 +72,8 @@ class MeshElementGrid final {
 		return elements_vec;
 	}
 
+	MathLib::Point3d const& getGridMinEdgePoint() const;
```
I'm not sure what "GridMinEdgePoint" means. can you add documentation or rename it?

bilke commented:
Jenkins test this please.

TomFischer commented:
@norihiro-w wrote:
> I'm not sure what "GridMinEdgePoint" means. can you add documentation or rename it?

Renamed it and added documentation, see commit f4be806.

norihiro-w commented:
:+1:

endJunction commented:
Jenkins, test this please!
